src/js/media-store/media-store.ts

```diff
@@ -527,7 +527,7 @@ export const createMediaStore = ({
           // responsible for managing/monitoring `defaultSubtitles` in the `defaultStateMediator`. (CJP)
           stateOwners.options[optionName] = optionValue;
         });
-        // updateStateFromFacade();
+        updateStateFromFacade();
```

Comment:
why was this needed?

Reply:
Yeah I think we need to take a step back here. Per out of band convo with @ronalduQualabs, this impacts other state change behaviors in the media store and also explicitly contradicts the code comment above. Will try to help with a strategy/rework here.

TL;DR answer to your question, @luwes - `mediaLang` is modeled as an "option". Options aren't designed to have "live" dynamic updates in the way that other modeled state is. We want `mediaLang` to have "live" dynamic updates. This does that (but is probably a "left turn instead of a right" in the solution).